### PR TITLE
Fix reactive counters when filtering on dates

### PR DESCRIPTION
**Pull request by vparpoil**
One test is failing for Reactive counts on named queries with date filtering (non reactive counts all work as expected). 
I couldn't find a fix for now, don't hesitate to give me some guidance on where to look for :) Thanks

See issue https://github.com/cult-of-coders/grapher/issues/401

### CODE REVIEW

File: lib/namedQuery/testing/client.test.js [hunk 1]
```diff
@@ -90,6 +106,24 @@ describe('Named Query', function() {
...
+        const handleCount = query.subscribeCount();
+        Tracker.autorun(c => {
+            if (handle.ready() && handleCount.ready()) {
+                c.stop();
+                const count = query.getCount();
+                const data = query.fetch();
+                handle.stop();
+                handleCount.stop();
+                assert.equal(data.length, 36);
+                assert.equal(count, 36);
```
Comment (vparpoil) [inline lib/namedQuery/testing/client.test.js]: Count here is always 0

**Comment by vparpoil** [comment]
I found the reason for this. 
The request is stringified [here](https://github.com/cult-of-coders/grapher/blob/842dbb1ed313ecd1abdd7fc79378be673445df1b/lib/query/counts/genEndpoint.server.js#L21) . Hence, the date object is transformed into a string and the mongo query fails to compare the string with dates.
Here is a proposition to fix this behavior using `meteor/ejson` instead of json

**Comment by vparpoil** [comment]
@StorytellerCZ I updated and resolved conflicts, thank you !

**Comment by StorytellerCZ** [comment]
Will be published in v1.4.1. Thank you @vparpoil!

**Pull request merged**